src/Nowruz/modules/layout/index.tsx

```diff
@@ -24,6 +25,7 @@ export const Layout = () => {
         <div className="w-full fixed top-0 right-0 z-20 bg-Base-White ">
           <HeaderNavBar setOpen={setOpen} logout={logOut} />
         </div>
+        <SearchModal open={true} onClose={() => console.log} />
```

Comment:
on close should accept undefined and do nothing for production use log is bad practice

Reply:
This is used for testing purposes for azin and shall be removed after test is passed.